src/controllers/api.controller.js

```diff
@@ -38,4 +40,8 @@ apiCtrl.reviews = async (req, res) => {
   res.json(await loadAllReviews());
 };
 
+apiCtrl.movieRating = async (req, res) => {
+  res.json(await loadMovieRatings(req.params.id));
+};
```

Comment:
I would have just done this in the `api.routes.js` module, where the handlers are used.